ast/src/parsed/build.rs

```diff
@@ -0,0 +1,43 @@
+use crate::parsed::{BinaryOperator, Expression, PolynomialReference};
+
+pub fn direct_reference<S: Into<String>, T>(name: S) -> Expression<T> {
+    Expression::PolynomialReference(PolynomialReference {
+        namespace: None,
+        name: name.into(),
+        index: None,
+        next: false,
+    })
+}
+
```

Comment:
This file looks duplicated?

Reply:
One is for the parsed ast, the other for the analyzed ast.